input/cql/PCSBMIScreenAndFollowUpFHIR.cql

```diff
@@ -131,6 +150,9 @@ define "Low BMI Interventions Ordered":
         )
   )
 
+/*
+@QICore6Uplift: Update 'Condition to 'ConditionProblemsHealthConcerns'
```

Comment:
Looks like a Procedure, so not sure about using Condition. No update required in my opinion.

Reply:
Further down in the code, there is a Condition resource type that is being used.